src/components/v5/common/ActionSidebar/partials/Motions/consts.ts

```diff
@@ -26,6 +26,6 @@ export const opposeOption = {
 };
 
 export const STAKING_RADIO_BUTTONS: ButtonRadioButtonItem<MotionVote>[] = [
-  supportOption,
   opposeOption,
+  supportOption,
```

Comment:
This branch will cause a merge conflict with #1394 

Just a heads up

Reply:
> As I understand it we're no longer doing prefix/suffix token symbol for the staking input.
> 
> @arrenv please confirm if that's still the case

Yes, that is still the case. It was never in the design as far as I understand.

Reply:
@arrenv @rdig There is token symbol on designs - https://www.figma.com/file/5V8pr7iMwXsT9L3VAZsmUt/Colony-V5?type=design&node-id=2729-220432&mode=design&t=wSu6J9exWt1VH7yU-0, or these are old ones?